experiments/ClimaEarth/leaderboard/leaderboard.jl

```diff
@@ -153,11 +160,151 @@ function compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
     CairoMakie.save(joinpath(leaderboard_base_path, "bias_leaderboard.png"), fig_leaderboard)
 end
 
+"""
+    compute_pfull_leaderboard(leaderboard_base_path, diagnostics_folder_path)
```

Comment:
I am using `pfull` to refer to full pressure levels since the variables have the dimensions longitude, latitude, time, and pressure levels. I am going to add it to the docstring, so it is more clear what the function needs.